tests/unit/Event/EventTest.php

```diff
@@ -66,9 +66,13 @@ public function testParams(): void
         $this->assertEquals('eventValue1', $this->object->getParam('eventKey1'));
         $this->assertEquals('eventValue2', $this->object->getParam('eventKey2'));
         $this->assertEquals(null, $this->object->getParam('eventKey3'));
-        global $register;
-        $pools = $register->get('pools');
-        $client = new Client($this->object->getQueue(), $pools->get('queue')->pop()->getResource());
+
+        global $global;
```

Comment:
why did we change the name from register / registry to $global? its confusing especially since we have a keyword with similar name